scripts/configuration/ac-build-config.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<config>
+    <sdkInfo>
+        <sdkVersion>1.0.2-SNAPSHOT</sdkVersion>
+        <iosVersion>1.0.0</iosVersion>
+        <androidVersion>1.0.0</androidVersion>
```

Comment:
what is that file and how versions are maintained here?

Reply:
iosVersion and androidVersion are the versions that we bind; sdkVersion is the version that we ship. The scripts are updated to change that value and read it instead of using a csproj file